### PR TITLE
add test for -last option

### PR DESCRIPTION
**Pull request by VladVega**
- Add test for `-last` option. 
- add bottom edge condition to the from option

fixes https://github.com/juttle/juttle-opentsdb-adapter/issues/18
fixes https://github.com/juttle/juttle-opentsdb-adapter/issues/12

### CODE REVIEW

**Comment by VladVega** [comment]
@davidvgalbraith

File: lib/read.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ let db = require('./db');
 let FilterCompiler = require('./filter');
 let Promise = require('bluebird');
 
+const VALDID_START_DATE_MIN = new Date(1000000000000).getTime();
```
Comment (davidvgalbraith) [inline lib/read.js]: What's a `VALDID`? Also `new Date(1000000000000).getTime()` is just `1000000000000` so you don't need the Date stuff at all.

**Comment by davidvgalbraith** [comment]
`-from :0:` is very useful, what's the story here that stops us from using it?

File: lib/read.js (hunk 1 above)
Comment (VladVega) [inline lib/read.js]: fixed.

**Comment by VladVega** [comment]
The story is that I'm not sure it's worth my time to investigate why a time under `new Date(1000000000000)` isn't valid based on this module: https://github.com/opentsdb-js/validate-time. 

Think it's worth it until we have a use case?

**Comment by davidvgalbraith** [comment]
I think anyone who wants to use this adapter is going to want to be able to `read -from :0:`. Whether we should do any work on an adapter no one uses is a reasonable question, but I think fixing `read -from :0:` is at least as valid as all the rest of the work we've done here. You can start your search for truth at https://github.com/opentsdb-js/validate-time/blob/master/lib/index.js#L43.

**Comment by VladVega** [comment]
Ok so let me tell you the rabbit hole that I knew this would be:
1. First off the drive had support for absolute formats: http://opentsdb.net/docs/build/html/user_guide/query/dates.html so instead of ms from epoch I tried passing absolute time. The problem: The query was spot on (driver was fine) but to opentsdb a from
`2015/02/29-11:11:07` to `2015/02/29-11:11:08` includes `2015/02/29-11:11:08.345` (milliseconds after 08s) but of course so does `2015/02/29-11:11:08` to `2015/02/29-11:11:09` WTF. So I decided not to use absolute time.

2. The opentsdb nodejs drivers are in several repos and most of them don't allow 0 for anything. So i needed to fork 4 repos in order to solve this problem. I created PRs for all the forks. In the meantime I added pointers to the branch in the package.json.

**Comment by davidvgalbraith** [comment]
Nice work!

**Comment by VladVega** [comment]
Is that a +1?

**Comment by davidvgalbraith** [comment]
+1
